src/modules/packagechooser/PackageChooserViewStep.cpp

```diff
@@ -188,6 +196,12 @@ PackageChooserViewStep::setConfigurationMap( const QVariantMap& configurationMap
         m_id = moduleInstanceKey().split( '@' ).last();
     }
 
+    m_description = ( configurationMap.contains( "description" ) ) ?
```

Comment:
Maybe we can get this from the empty-id item if it's there, rather than from a new key? It seems to me that you're taking the description from the `.conf` file too literally and that we can make "" work for all kinds of module description.